pygeos/io.py

```diff
@@ -172,37 +181,59 @@ def from_wkt(geometry, **kwargs):
     ----------
     geometry : str or array_like
         The WKT string(s) to convert.
+    on_invalid : str, one of {"raise", "warn", "ignore"} (default: "raise")
```

Comment:
From https://numpydoc.readthedocs.io/en/latest/format.html:

> When a parameter can only assume one of a fixed set of values, those values can be listed in braces, with the default appearing first:
> order : {'C', 'F', 'A'}
    Description of `order`.

```suggestion
    on_invalid : {"raise", "warn", "ignore"}
```

The default is also listed in the call signature, which is visible in the docs.

Also the markup of the list below is not ok, see https://pygeos--321.org.readthedocs.build/en/321/io.html#pygeos.io.from_wkb. You might want each line with a hyphens (not sure)